### PR TITLE
Patrick j govalpa patch 1

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
@LongForWisdom @Gala-G I've started a consistency pass.

I've started with surplus auction parameters. I'll keep moving through different parameters. Let me know if you spot anything else that should be fixed.

### CODE REVIEW

**Comment by patrick-j-govalpha** [comment]
I've now gone through the debt auction docs too

File: governance/weekly-governance-cycle.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ The Weekly Governance Cycle works in conjunction with the Monthly Governance Cyc
...
+- Ratified Facilitators (a list can be found [here](https://mips.makerdao.com/mips/details/MIP38#MIP38c2)) are authorized by Maker Governance to interact with the Weekly Cycle. Facilitators may submit Non-Standard Weekly Polls that are related to their Core Unit Mandat. If necessary, and if it is related to their Core Unit Mandate, Facilitators can skip the Non-Standard Weekly Polls and include logic directly into the weekly Executive Vote.
```
Comment (Gala-G) [inline governance/weekly-governance-cycle.md:15]: missing e: Mandate

File: parameter-index/surplus-auction/param-bid-duration-flap.md [hunk 2]
```diff
@@ -24,32 +24,32 @@ Surplus Auctions are used to auction off excess DAI in fixed lots for a variable
...
 
-A larger `Bid Duration (Flap)` gives keepers more time to participate in auctions, hopefully encouraging a higher number of bidders. If the `Bid Duration (Flap)` is too large, there may be bids for minimal amounts of MKR to safeguard against price volatility during the `Bid Duration (Flap)` period. Realistically priced bids would only appear when the auction end (determined by Auction Duration) is closer than the `Bid Duration (Flap)` period. In situations where the price of MKR is dropping, this would lead to less MKR being burned than with a smaller `Bid Duration (Flap)`.
+A larger Bid Duration (Flap) gives keepers more time to participate in auctions, hopefully encouraging a higher number of bidders. If the `Bid Duration (Flap)` is too large, there may be bids for minimal amounts of MKR to safeguard against price volatility during the Bid Duration (Flap) period. Realistically priced bids would only appear when the auction end (determined by Auction Duration) is closer than the Bid Duration (Flap) period. In situations where the price of MKR is dropping, this would lead to less MKR being burned than with a smaller Bid Duration (Flap).
```
Comment (Gala-G) [inline parameter-index/surplus-auction/param-bid-duration-flap.md:36]: there is still one `Bid Duration (Flap)` left - second sentence after "If the..."

File: parameter-index/debt-auction/param-bid-duration-flop.md [hunk 3]
```diff
@@ -23,31 +23,37 @@ Debt Auctions are used to recapitalize the system by minting and auctioning off
...
 
-A larger `Bid Duration (Flop)` gives keepers more time to participate in auctions, hopefully encouraging a higher number of bidders. If the `Bid Duration (Flop)` is too large, there may be bids for very high amounts of MKR to safeguard against price volatility during the `Bid Duration (Flop)` period. Realistically priced bids would only appear when the auction end (determined by Auction Duration) is closer than the `Bid Duration (Flop)` period. In situations where the price of MKR is dropping, this would lead to more MKR being minted than with a smaller `Bid Duration (Flop)`.
+A larger Bid Duration (Flop) gives keepers more time to participate in auctions, hopefully encouraging a higher number of bidders. If the Bid Duration (Flop) is too large, there may be bids for very high amounts of MKR to safeguard against price volatility during the `Bid Duration (Flop)` period. Realistically priced bids would only appear when the auction end (determined by Auction Duration) is closer than the Bid Duration (Flop) period. In situations where the price of MKR is dropping, this would lead to more MKR being minted than with a smaller Bid Duration (Flop).
```
Comment (Gala-G) [inline parameter-index/debt-auction/param-bid-duration-flop.md:35]: There is still one `Bid Duration (Flop)` at the end of the second sentence.

**Review by Gala-G — CHANGES_REQUESTED** [review]

File: governance/weekly-governance-cycle.md [hunk 4]
```diff
@@ -11,8 +11,8 @@ The Weekly Governance Cycle works in conjunction with the Monthly Governance Cyc
...
```
Comment (Gala-G) [inline governance/weekly-governance-cycle.md:15]: missing e: Mandate

**Review by Gala-G — CHANGES_REQUESTED** [review]

File: parameter-index/debt-auction/param-auction-duration-flop.md [hunk 5]
```diff
@@ -45,6 +45,10 @@ Maker Governance may wish to decrease the Auction Duration (Flop) if the auction
 ## Considerations
 Auction Duration (Flop) is always lower bounded by the Bid Duration (Flop) i.e. the time period before which a bid expires. If it is set lower than the Bid Duration (Flop), it will render that parameter unnecessary since the auction will end before a bid expires. 
 
-For example, if Bid Duration (Flop) is 1 hour and Auction Duration (Flop) is 30 minutes, then the auction will only last 30 minutes, making the Bid Duration (Flop) irrelevant.
+### Example
+`Bid Duration (Flop)` = 1 hour  
+`Auction Duration (Flop)` = 30 minutes
+
+The the auction will only last 30 minutes, making the Bid Duration (Flop) irrelevant.
```
Comment (Gala-G) [inline parameter-index/debt-auction/param-auction-duration-flop.md]: Did you mean "Then the auction" ? Erase repetition "The the"

**Review by Gala-G — CHANGES_REQUESTED** [review]

**Comment by Gala-G** [comment]
Hey Patrick,

They seem pretty good to my non-technical eye. I spotted some typos and addressed them in my review. 
Let's wait for @LongForWisdom to review it and merge it.

thanks!
g

File: parameter-index/debt-auction/param-bid-duration-flop.md (hunk 3 above)
Comment (patrick-j-govalpha) [reply parameter-index/debt-auction/param-bid-duration-flop.md]: fixed

File: parameter-index/surplus-auction/param-bid-duration-flap.md (hunk 2 above)
Comment (patrick-j-govalpha) [reply parameter-index/surplus-auction/param-bid-duration-flap.md]: fixed

File: governance/weekly-governance-cycle.md (hunk 4 above)
Comment (patrick-j-govalpha) [reply governance/weekly-governance-cycle.md]: fixed

**Comment by patrick-j-govalpha** [comment]
updated with feedback from @Gala-G

**Pull request merged**